MobileWallet/TariLib/wallet.h

```diff
@@ -391,6 +391,7 @@ struct TariCommsConfig *comms_config_create(const char *public_address,
                                      const char *datastore_path,
                                      unsigned long long discovery_timeout_in_secs,
                                      unsigned long long saf_message_duration_in_secs,
+                                     const char *network,
```

Comment:
Is that change require us to update the FFI library? If yes, then we need to deploy new binary on AWS S3 and update the `update_dependencies` script settings.

Reply:
Yes, it is a breaking change. This PR will allow you to be able to select the network the wallet connects to from the UI.